jdcoefct.c

```diff
@@ -584,18 +590,20 @@ decompress_smooth_data(j_decompress_ptr cinfo, JSAMPIMAGE output_buf)
           DC20 = (int)next_block_row[2][0];
           DC25 = (int)next_next_block_row[2][0];
         }
-        /* Compute coefficient estimates with a Gaussian-like kernel,
+        /* Compute coefficient estimates with some smoothing. Here the
+         * coefficients come from smoothing with Gaussian like kernel,
```

Comment:
Why is this extra wording necessary?  Please explain what you're trying to convey so that I can appropriately wordsmith it.

Reply:
not necessary, keep as you suggested: 
       /* Compute coefficient estimates with a Gaussian-like kernel,
         * keeping the averages of the DC values.

Reply:
(probably came from a botched merge..)